Fix get device information when the application has partial trust mode (#1432)
microsoft/appcenter-sdk-dotnet: CHANGELOG.md

#### WPF/WinForms

* **[Fix]** Fix exception handling when the app tries to access a locked database.

> * **[Fix]** Fix get device information when the application has partial trust mode.

Reviewer: ```suggestion
* **[Fix]** Fix getting device information when the application runs in partial trust mode.
```
